File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for(int i=0;i<nums.length;i++){
            for(int j=i+1;j<nums.length;j++){
                if(nums[i] + nums[j] == target){
                    int[] array = new int[2];
                    array[0] = i;
                    array[1] = j;
                    return array;
                }
            }
        }
        return null;
    }
}

File: 0001-two-sum/0001-two-sum.rs
impl Solution {
    pub fn two_sum(nums: Vec<i32>, target: i32) -> Vec<i32> {

        let mut index1 : usize = 0;
        let len = nums.len();

        while index1 < len {
            let mut index2 = index1 + 1;
            while index2 < len {
                if nums[index1] + nums[index2] == target {
                    return vec![index1 as i32, index2 as i32];
                }
                index2 += 1;
            }
            index1 += 1;
        }

        return vec![0,0]
    }
}